src/dragon_eval/evaluation.py

```diff
@@ -334,6 +383,24 @@ def score(self, *, task_name: str, job_name: str):
                 average="weighted",
             )
 
+        elif TASK_TYPE[task_name] == EvalType.TEXT_TARGET:
+            # evaluate single-label named entity recognition with tags inserted in the text tasks
+            # metric: blended redaction F1 score
+            if y_orig is None:
+                raise ValueError(
+                    f"Could not find text column for {label_column} (job: {job_name})"
```

Comment:
Could you indicate both keys here? For example "Could not find original text at {..} for label {..} (job: ...)"